commands/channel_e2e_test.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"fmt"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+	"github.com/spf13/cobra"
```

Comment:
```suggestion
	"fmt"

	"github.com/spf13/cobra"
	
	"github.com/mattermost/mmctl/client"
	"github.com/mattermost/mmctl/printer"
```

Fix for the CI lint error.

Reply:
whoops
thanks, fixed!